projects/ad9083_evb/common/ad9083_evb_bd.tcl

```diff
@@ -43,18 +37,17 @@ adi_tpl_jesd204_rx_create rx_ad9083_tpl_core $RX_NUM_OF_LANES \
                                                $RX_SAMPLES_PER_FRAME \
                                                $RX_SAMPLE_WIDTH
 
-ad_adcfifo_create $adc_fifo_name $adc_data_width $adc_dma_data_width $adc_fifo_address_width
-
 ad_ip_instance axi_dmac axi_ad9083_rx_dma [list \
-  DMA_TYPE_SRC 1 \
+  DMA_TYPE_SRC 2 \
   DMA_TYPE_DEST 0 \
   CYCLIC 0 \
   SYNC_TRANSFER_START 0 \
   DMA_2D_TRANSFER 0 \
   MAX_BYTES_PER_BURST 4096 \
   AXI_SLICE_DEST 1 \
   AXI_SLICE_SRC 1 \
-  DMA_LENGTH_WIDTH 24 \
+  FIFO_SIZE 32\
+  DMA_LENGTH_WIDTH 31 \
   DMA_DATA_WIDTH_DEST $adc_dma_data_width \
```

Comment:
It would make sense to set the DMA width of the destination to match the PS AXI slave bus size (128) to avoid insertion of conversion logic.

Reply:
Updated destination to 128bits. Tested in hardware